product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -6,30 +6,30 @@ deepToC: true
 
 ## Commit scope
 
-[Commit scopes](../../../pgd/latest/durability/commit-scopes/) in PGD are a set of rules that describe the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they have an effect on consistency, durability, and performance.
+[Commit scopes](/pgd/latest/durability/commit-scopes/) in PGD are a set of rules that describe the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they have an effect on consistency, durability, and performance.
 
-The actual behavior depends on which kind of commit scope a commit scope's rule uses: [Group Commit](/../../../pgd/latest/durability/group-commit/), [Commit At Most Once](/../../../pgd/latest/durability/camo/), [Lag Control](/../../../pgd/latest/durability/lag-control/), [Synchronous Commit](/../../../pgd/latest/durability/synchronous_commit/), or a combination of these.
+The actual behavior depends on the kind of commit scope a commit scope's rule uses: [Group Commit](/pgd/latest/durability/group-commit/), [Commit At Most Once](/pgd/latest/durability/camo/), [Lag Control](/pgd/latest/durability/lag-control/), [Synchronous Commit](/pgd/latest/durability/synchronous_commit/), or a combination of these.
```

Comment:
You do not need to (and should not) use relative paths when referring to a different doc set. The above construction is correct.